etl/steps/data/garden/unicef/2024-07-30/child_migration.meta.yml

```diff
@@ -0,0 +1,115 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Migration
+        - Violence Against Children & Children's Rights
+    unit: people
+    short_unit: ""
+    processing_level: minor
+  unicef_descriptions:
+    refugee: |-
+      Person who are outside their country of nationality or habitual residence, who cannot return due to a well-founded fear of being persecuted because of their race, religion, nationality, membership of a particular social group or political opinion. This number only accounts those who have been recognized as refugees or find themselves in refugee-like situations.
+      This data is sourced from: *United Nations High Commissioner for Refugees, Global Trends: Forced Displacement in 2018, UNHCR, Geneva, 2019.* Share of under 18 from UNHCR unpublished data, cited with permission.
+    internally_displaced: |-
+      Persons who have been forced or obliged to flee or leave their homes or places of habitual residence, in particular as a result of or in order to avoid the effects of armed conflict, situations of generalized violence, violations of human rights or natural or human-made disasters, and who have not crossed an internationally recognized border. Data presented in this table refer only to persons displaced due to conflict and violence.
+      This data is sourced from: *Internal Displacement Monitoring Centre, Global Report on Internal Displacement 2019 (GIDD), IDMC, Geneva, 2019.*
+    international_migrant: |-
+      Persons living in a country or area other than their country of birth.
+      This data is sourced from: *United Nations, Department of Economic and Social Affairs, Population Division, Trends in International Migrant Stock: The 2019 Revision, United Nations, New York, 2017.* Share of under 18 calculated by UNICEF based on *United Nations, Department of Economic and Social Affairs, Population Division, Trends in International Migrant Stock: Migrants by Age and Sex, United Nations, New York, 2019.*
+  other_descriptions:
+    unrwa: |-
+      The United Nations Relief and Works Agency for Palestine Refugees in the Near East (UNRWA) is a UN agency that was established in 1949 to carry out direct relief and works programmes for Palestine refugees.
+      UNRWA has a humanitarian and development mandate to “provide assistance and protection to Palestine refugees pending a just and lasting solution to their plight.” This includes essential service delivery, primarily in the areas of basic education, primary health care and mental health care, relief and social services, microcredit, and emergency assistance, including in situations of armed conflict to registered Palestinian refugees.
+      They operate in Jordan, Lebanon, Syria the West Bank including East Jerusalem, and Gaza. The Agency does not have a mandate to engage in political negotiations or durable solutions.
```

Comment:
They operate in Jordan, Lebanon, Syria***,*** the West Bank***,*** including East Jerusalem, and Gaza. The Agency does not have a mandate to engage in political negotiations or durable solutions.